tests/rptest/archival/abs_client.py

```diff
@@ -64,7 +65,12 @@ def _wait_no_key(self, blob_client: BlobClient, timeout_sec: float = 10):
 
     def create_bucket(self, name: str):
         blob_service = BlobServiceClient.from_connection_string(self.conn_str)
-        blob_service.create_container(name)
+        try:
+            blob_service.create_container(name)
+        except ResourceExistsError as e:
+            # ignore if we get ResourceExistsError:
+            #     The specified container already exists
+            pass
```

Comment:
I am not too convinced here but if we do this, can validate that the container does exists as expected and can be used by the test? Does it need to be cleaned?

Reply:
I've abandoned this approach because I think I found (and fixed) the actual cause of the issue.